src/main/resources/assets/actuallyadditions/lang/en_US.lang

```diff
@@ -579,6 +581,10 @@ tooltip.actuallyadditions.battery.changeMode=Sneak-right-click to toggle.
 tooltip.actuallyadditions.previouslyDoubleFurnace=Previously "Double Furnace"
 tooltip.actuallyadditions.previouslyBag=Previously "Bag"
 tooltip.actuallyadditions.previouslyVoidBag=Previously "Void Bag"
+tooltip.actuallyadditions.item_filling_wand.selectedBlock=Selected Block: %s
+tooltip.actuallyadditions.item_filling_wand.selectedBlock.none=None
+tooltip.actuallyadditions.itemEnergy=%s/%s Crystal Flux
```

Comment:
Well, the french translation for "Crystal Flux" is "Flux Cristal",  hardcoding it will make it look wierd, as all other names are translated.
I guess that you don't want the energy unit to be changed by resource packs and you're right